manifests/operator/kustomization.yaml

```diff
@@ -14,5 +14,4 @@
 
 resources:
 - ../base
-- ../hierarchyconfig-crd.yaml
```

Comment:
Please also remove the config/tooling for generating this file, since it is no longer used

Reply:
Updated `Makefile.reconcilermanager`. Any other locations I am missing?

Reply:
You'll also need to update `manifests/patch/kustomization.yaml` and update the kubebuilder comments at `pkg/api/configmanagement/v1/types.go` to not generate a CRD.

Reply:
How does it look now?

Reply:
Left a comment on the api struct. Also should delete the hierarchconfig-crd.yaml file.

Reply:
Fixed it.